app/ui/lib/SideModal.tsx

```diff
@@ -53,7 +53,6 @@ export function SideModal({
   animate = true,
   errors,
 }: SideModalProps) {
-  const titleId = 'side-modal-title'
```

Comment:
This is only need to hook up the accessible title to the dialog, but Radix takes care of that for us.

<img width="578" alt="image" src="https://github.com/user-attachments/assets/ce752b06-6b9d-49e1-a905-3aaa5b672f45">